### ISSUE TITLE
Performance: Improve BatchNormalization forward/backward to align with oneDNN implementation.

### ISSUE THREAD

**Issue by fengyuan14** [intel/torch-xpu-ops#937]
### 🚀 The feature, motivation and pitch

Basing on the consideration of accuracy, we followed the PyTorch CUDA implementation, using Welford algorithm and similar kernel template. Will improve the kernel template with vectorized load/store.

**Comment by fengyuan14** [comment]
https://github.com/intel/torch-xpu-ops/pull/933

**Comment by xytintel** [comment]
Merged
